vector/src/main/java/im/vector/riotx/features/login/LoginViewState.kt

```diff
@@ -73,7 +73,7 @@ data class LoginViewState(
             append(homeServerUrl?.trim { it == '/' })
             append(SSO_REDIRECT_PATH)
             // Set a redirect url we will intercept later
-            appendParamToUrl(SSO_REDIRECT_URL_PARAM, RIOTX_REDIRECT_URL)
+            appendParamToUrl(SSO_REDIRECT_URL_PARAM, VECTOR_REDIRECT_URL)
```

Comment:
Any particular reason this is `VECTOR` rather than `ELEMENT`?

Reply:
I prefer to use Vector internally